configure.ac

```diff
@@ -410,6 +410,25 @@ AC_ARG_WITH(yang-standard-dir,
 	    )
 AC_MSG_RESULT(Standard YANG files expected to be in ${YANG_STANDARD_DIR})	
 
+# who we run as
```

Comment:
These settings are for examples and tests only, please add that in comment.
Document the configure options also in https://clixon-docs.readthedocs.io/en/latest/install.html#configure-options

Reply:
Done